common/lib/dependabot/file_fetchers/base.rb

```diff
@@ -42,11 +42,14 @@ def self.required_files_message
       # If provided, file _data_ will be loaded from the clone.
       # Submodules and directory listings are _not_ currently supported
       # by repo_contents_path and still use an API trip.
-      def initialize(source:, credentials:, repo_contents_path: nil)
+      #
+      # options supports custom feature enablement
+      def initialize(source:, credentials:, repo_contents_path: nil, options: {})
         @source = source
         @credentials = credentials
         @repo_contents_path = repo_contents_path
         @linked_paths = {}
+        @options = {}
```

Comment:
Did you mean to actually set it or is it deliberate to keep in empty for now?

Reply:
No this was a typo, thanks 😄